aws/resource_aws_security_group_test.go

```diff
@@ -1599,6 +1601,13 @@ func testAccCheckAWSSecurityGroupAttributesChanged(group *ec2.SecurityGroup) res
 				group.IpPermissions[0], group.IpPermissions[1]
 		}
 
+		if len(group.IpPermissions[1].IpRanges) > 1 {
```

Comment:
I think this could probably just be rewritten as a sort on the `IpRanges` if its necessary instead of this very explicit swap.

Reply:
Yeah, I really just copied the block above this one that does exactly same thing with IpPermissions. Although sort might be more universal, it does not add much value here and may be more expensive computationally.